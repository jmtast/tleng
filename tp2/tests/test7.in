// Error de sem√°ntica: 0 repeticiones por minuto
#tempo negra 0
#compas 2/4

const oct1 = 5;

voz (0)
{
  compas
  {
    silencio(blanca);
  }
}
