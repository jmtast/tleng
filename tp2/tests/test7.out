Error:
Cantidad de repeticiones por minuto incorrecta. Debe ser mayor a 0. LÃ­nea 2
